runtime/src/main/java/io/quarkiverse/fury/FuryBuildTimeConfig.java

```diff
@@ -17,6 +19,40 @@ public interface FuryBuildTimeConfig {
     @WithDefault("true")
     boolean requiredClassRegistration();
 
+    /** Whether track shared or circular references. */
+    @WithDefault("false")
+    boolean trackRef();
+
+    /**
+     * Set class schema compatible mode.
+     * <br>
+     * SCHEMA_CONSISTENT: Class schema must be consistent between serialization peer and deserialization peer.
+     * <br>
+     * COMPATIBLE: Class schema can be different between serialization peer and deserialization peer. They can
+     * add/delete fields independently.
+     */
+    @WithDefault("SCHEMA_CONSISTENT")
+    CompatibleMode compatibleMode();
```

Comment:
`CompatibleMode` is enum? I'm not very sure that it is supported here.

Reply:
Yep, it's an enum